### ISSUE TITLE
Labels do not wrap (or cut) after drilldown

### ISSUE THREAD

**Issue by balkarov** [highcharts/highcharts#9218]
Labels do not wrap (or cut) after drilldown

![1](https://user-images.githubusercontent.com/6030335/47210229-0f9ae080-d39b-11e8-8d04-3f86b60967ee.gif)

#### Expected behaviour
labels words wrap and columns show normal

#### Actual behaviour
labels show full name and columns not visible

#### Live demo with steps to reproduce
https://jsfiddle.net/ogfrLphv/

#### Product version
Highcharts version 6.1.4

#### Affected browser(s)
Chrome 69.0.3497.100

**Comment by balkarov** [comment]
@pawelfus hi! 
Thank you for your workaround👍

**Comment by balkarov** [comment]
@pawelfus you hack broke chart before drilldown
https://jsfiddle.net/zfseyh75/

**Comment by pawelfus** [comment]
Hi @balkarov 

Thank you for testing my solution! You are right, it breaks something else. 

**Workaround:**
- https://jsfiddle.net/BlackLabel/ogfrLphv/11/
- http://jsfiddle.net/BlackLabel/d7f5pq3z/26/

Snippet:
```javascript
(function(H) {

  H.wrap(H.Axis.prototype, 'renderUnsquish', function(proceed) {
    var slotWidth = this.getSlotWidth();

    proceed.apply(this, Array.prototype.slice.call(arguments, 1));

    H.each(this.tickPositions, function(pos) {
      var tick = this.ticks[pos];

      if (
        tick &&
        tick.label &&
        tick.label.styles &&
        !tick.label.styles.width &&
        slotWidth &&
        tick.label.getBBox().width > slotWidth
      ) {
        tick.label.css({
          width: slotWidth
        });
      }
    }, this);
  });
})(Highcharts);
```

*Internal note:*
It looks like resetting width should be applied only for ellipsis, not for word-wrapping (related to #8210), but I may be wrong. 

Related lines:
https://github.com/highcharts/highcharts/blob/bd59aa5b1145d1dc8e85f57f43df38a8e92bf3d3/js/parts/Axis.js#L5565-L5572

**Comment by balkarov** [comment]
@pawelfus thank you for your help!

**Comment by balkarov** [comment]
@pawelfus hello!
Sorry but your hack broke another case
http://jsfiddle.net/tL4aoh8n/

**Comment by pawelfus** [comment]
Right! Exclude rotated labels. Demo: http://jsfiddle.net/BlackLabel/tL4aoh8n/3/

**Comment by balkarov** [comment]
@pawelfus thanks a lot!

**Comment by balkarov** [comment]
@pawelfus hello.
I found bug with your last workaround
https://jsfiddle.net/j7a1c4gv/1/

![image](https://user-images.githubusercontent.com/6030335/104431623-24c24200-5599-11eb-9870-3455a6fb94c5.png)


But if I will delete `!tick.rotation &&` it's working
![image](https://user-images.githubusercontent.com/6030335/104431764-5509e080-5599-11eb-9067-cf483aa65e80.png)



But if delete  `!tick.rotation &&` it's working, does not work this case 
http://jsfiddle.net/tL4aoh8n/

**Comment by PaulDalek** [comment]
Hi @balkarov

From what I’ve checked the initial issue reported some time ago doesn’t occur in the newest version (it was fixed in v7.1.2) so you don’t need the workarounds anymore. As for your latest report, this behavior (decreasing the chart’s height on drilldown) looks like a different bug. Could you please open a new issue for it? Thanks.
